man/udevadm.xml

```diff
@@ -175,6 +175,13 @@
             <para>Cleanup the udev database.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-w</option></term>
+          <term><option>--wait-for-initialization<optional>=SECONDS</optional></option></term>
```

Comment:
We also have `udevadm trigger -w/--settle`. Those two options have similar meaning, so I guess that using `-w` for both of them is reasonable.